contracts/HashRegistrarSimplified.sol

```diff
@@ -381,7 +416,7 @@ contract Registrar {
         Deed bid = sealedBids[bidder][seal];
         // If the bid hasn't been revealed after any possible auction date, then close it
         if (address(bid) == 0
-            || now < bid.creationDate() + initialAuctionPeriod) throw;
+            || now < bid.creationDate() + launchLength) throw;
```

Comment:
Making it auctionLength + 2 weeks. (auction length already includes the reveal length